scripts/bytecodecompare/prepare_report.py

```diff
@@ -26,6 +26,11 @@ class CompilerInterface(Enum):
     STANDARD_JSON = 'standard-json'
 
 
+class ExecutionArchitecture(Enum):
+    NATIVE = 'native'
+    X86_64 = 'osx_intel'
```

Comment:
Let's keep those two in sync:
```suggestion
    OSX_INTEL = 'osx_intel'
```
Making it sound generic, like x86_64 is misleading anyway. `arch` is a macOS-only thing, isn't it?